src/components/layout/navbars/Navbar.tsx

```diff
@@ -391,7 +403,6 @@ export default function Navbar(props: Readonly<NavbarProps>) {
                       >
                         {props.textContent.links.login}
                       </a>
-                      <LanguageMobileBox darkMode={props.darkMode} singlesDay={props.singlesDay} />

```

Comment:
You removed the language selector for mobile view. Is it ok?

Reply:
No, fixed